src/nile/core/account.py

```diff
@@ -77,16 +82,59 @@ def deploy(self):
 
         return address, index
 
-    def send(self, address_or_alias, method, calldata, max_fee, nonce=None):
-        """Execute a tx going through an Account contract."""
-        if not is_alias(address_or_alias):
-            address_or_alias = normalize_number(address_or_alias)
+    def declare(
```

Comment:
In this release, we are adding query calls to the `invoke` command. We should add the query flags to the `declare` command too (**estimate_fee** and **simulate**). Even if it is not part of this PR, I think it would be good to include it, at least in the next release.

Reply:
created https://github.com/OpenZeppelin/nile/issues/242